packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -278,7 +311,10 @@ const ruleForBabelLoader = (babelPlugins) => {
     return {
         id: 'babel-loader',
         test: /(\.js(x?)|\.ts(x?))$/,
-        exclude: /node_modules/,
+        ...(pkg?.ccExtensibility?.overridesDir && pkg?.ccExtensibility?.extends
+            ? // TODO: handle for array here when that's supported
+              {exclude: makeRegExp(`/node_modules(?!/${pkg?.ccExtensibility?.extends})`)}
+            : {exclude: /node_modules/}),
```

Comment:
don't double-compile via `extends` files that will be compiled later in the pipeline via babel